core/src/main/java/org/mobilitydata/gtfsvalidator/parsing/RowParser.java

```diff
@@ -157,39 +159,38 @@ public String asEmail(int columnIndex, boolean required) {
    *     unknown, only phone number starting by "+" are validated.
    */
   @Nullable
-  public String asPhoneNumber(int columnIndex, boolean required) {
-    return asValidatedString(columnIndex, required, fieldValidator::validatePhoneNumber);
+  public String asPhoneNumber(int columnIndex, FieldLevelEnum level) {
```

Comment:
We will need to change the doc string here. I think it makes the Task :aggregateJavadoc to fail [here](https://github.com/MobilityData/gtfs-validator/runs/6619597119?check_suite_focus=true) and [here](https://github.com/MobilityData/gtfs-validator/runs/6619597193?check_suite_focus=true).